src/types.db

```diff
@@ -249,6 +249,7 @@ slurm_last_cycle_depth  value:GAUGE:0:U
 slurm_cycle_depth       value:DERIVE:0:U
 slurm_job_stats         value:DERIVE:0:U
 slurm_queue_length      value:DERIVE:0:U
+slave_running           value:GAUGE:0:U
```

Comment:
We're aiming at metric types that are a bit less specific than "slave running" (though, admittedly, we haven't consistently done a great job in the past). My recommendation would be to add a new type `bool value:GAUGE:0:1` and report these metrics as:

*   bool-slave-sql-running
*   bool-slave-io-running

Reply:
Okay, will push this change to my fork later today.